### ISSUE TITLE
Navigator Layer Generator to SVG generates with all techniques in all columns

### ISSUE THREAD

**Issue by jaimeatwork** [mitre-attack/mitreattack-python#33]
Hello, 

Thanks for all the work you put into this. I am running into an issue and not sure if the problem is what I'm doing or something else. 

When I try to use the CLI tool or the python examples you provide in the repo, I am generating strange looking svgs. With a json layer file that looks like this:

```
{
    "versions": {
        "attack": "9.1",
        "navigator": "4.2",
        "layer": "4.2"
    },
    "layout": {
        "layout": "side",
        "showName": true,
        "showID": true
    },
    "showTacticRowBackground": true,
    "tacticRowBackground": "#dddddd",
    "selectTechniquesAcrossTactics": true,
    "selectSubtechniquesWithParent": true,
    "name": "sample_layer",
    "description": "sample description",
    "domain": "enterprise-attack",
    "techniques": [
        {
            "techniqueID": "T1580",
            "score": 10
        }
    ],
    "gradient": {
        "colors": [
            "#ffffff",
            "#004f92"
        ],
        "minValue": 0,
        "maxValue": 10
    }
}
```

I run `layerExporter_cli layer.json -m svg -o demo.svg` and get an SVG that looks like the following where every column includes every technique.

![image](https://user-images.githubusercontent.com/84590541/134360517-8eecdb12-5e7c-4f75-ba0e-3fc74cf11634.png)

I also tried python code pieced together from your examples like this

```
import json
from mitreattack.navlayers import Layer, ToSvg, SVGConfig

with open('layer.json', 'r') as f:
    data = json.loads(f.read())

lay = Layer()
lay.from_dict(data) ## I tried both from_file and from_dict

y = SVGConfig(width=8.5, height=11, headerHeight=1, unit="in", showSubtechniques="expanded",
                 font="sans-serif", tableBorderColor="#6B7279", showHeader=True, legendDocked=True,
                 legendX=0, legendY=0, legendWidth=2, legendHeight=1, showLegend=True, showFilters=True,
                 showAbout=True, showDomain=True, border=0.104)

t = ToSvg(domain=lay.layer.domain, source='taxii', local=None, config=y)
t.to_svg(layerInit=lay, filepath='demo2.svg')
```

I want to eventually convert the svg to a png and embed it somewhere, but am not sure what I am missing. Thanks!

**Comment by ArtificialErmine** [comment]
Hey @jamieatwork, thanks for pointing this out! I can confirm that there is some kind of bug going on here, thought I'm not entirely sure exactly what is at fault just yet. I'll start digging into it and see if I can patch it.

**Comment by ArtificialErmine** [comment]
@jaimeatwork The bug should be fixed in the recently released v1.3.1. Thanks for pointing the issue out to us!

If you're curious as to the root/specifics of the issue, the v2.3.0 update to `taxii2-client` module broke certain functionality requirements related to filtering (https://github.com/oasis-open/cti-taxii-client/issues/104), resulting in a cascading issue that made it look like all tactics had every technique in the matrix. Locally, I just rolled back the version used by `mitreattack-python` to the last known good version.

**Comment by jaimeatwork** [comment]
Excellent! Thanks! This change worked out perfectly, thanks for getting to it so quickly.
